.github/workflows/generate-release-crs.yaml

```
@@ -0,0 +1,96 @@
+name: Apply Konflux Override Snapshot and create Release CR PR
+on:
+  workflow_dispatch:
+    inputs:
+      revision:
+        description: "SO branch or revision with the manifests"
+        default: "main"
```

```suggestion
        default: "release-1."
```

maybe that's a better default so that I only need to type the minor version ?